lab7_andrewdefran

```diff
@@ -0,0 +1,90 @@
+#Andrew DeFran
+#2/29/2020
+#lab7
+#Itauma
+
+#import statements
+import math
+ 
+#global variables
+
+#functions
+#user input function
+def userInput():
+    userin = int(input("please enter an integer greater than zero: "))
+
+    #validation check
+    while (userin < 0 ):
+        print("\nError: invalid input") 
+        userin = int(input("please enter an integer greater than zero: "))
+
+    return userin
+
+#problem 1: write a function that takes positive integer input and returns
+#it's factors as a positive array.
+def factorsOfNumber(a):
+
+    #blank array
+    array = []
+
+    #loop up to the number and check if number divides then add factor to array
+    for i in range(1, a + 1):
+        if (a % i == 0):
+            array.append(i)
```

Comment:
Review the pseudocode given. You didn't implement step b (floor of the square root of the number).